distributed/distributed.yaml

```diff
@@ -12,6 +12,9 @@ distributed:
     allowed-failures: 3     # number of retries before a task is considered bad
     bandwidth: 100000000    # 100 MB/s estimated worker-worker bandwidth
     default-data-size: 1000
+    # Number of seconds to wait until workers or clients are removed from the events log
+    # after they have been removed from the scheduler
+    events-cleanup-delay: 3600
```

Comment:
I recommend that we use `"1h"` or `"1 hour"` here and make sure that we use `distributed.utils.parse_timedelta` wherever it is read.